### PR TITLE
feat: move to ts, add icons in mainmenu

### LINKED ISSUE #162: show item icons in nav
To make navigation clear, we should by default show the item type icon (folder, document, app, etc.) in the navigation menu on the left.

In the future, this icon could be modifiable.

### PR DESCRIPTION
**Pull request by pyphilia**
close #162

### CODE REVIEW

**Comment by pyphilia** [comment]
@spaenleh this branch is currently on dev if you want to test. I had some strange onclick problem (in the sidebar), let me know if it's alright for you.

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-          start: yarn start:ci
-          wait-on: 'http://localhost:3000'
-          wait-on-timeout: 180
-          browser: chrome
-          quiet: true
+# This workflow is made up of one job that calls the reusable workflow in graasp-deploy
+jobs:
+  graasp-deploy-cypress-workflow:
+    # Replace with repository name
+    name: Graasp Perform
```
Comment (spaenleh) [inline .github/workflows/cypress.yml:15]:
```suggestion
    name: Graasp Player
```

File: cypress/e2e/main.cy.js [hunk 2]
```diff
@@ -109,7 +109,7 @@ describe('Main Screen', () => {
   });
 
   describe('Use cases', () => {
-    it(`Display ${STATIC_ELECTRICITY.items[0].name}`, () => {
+    it.only(`Display ${STATIC_ELECTRICITY.items[0].name}`, () => {
```
Comment (spaenleh) [inline cypress/e2e/main.cy.js:112]: remove only ?

**Review by spaenleh — APPROVED** [review]
🍿Looks good to me ! Just a few comments but you can merge once they are resolved !

**Pull request merged**